### ISSUE TITLE
Minimum set of files to calculate core?

### ISSUE THREAD

**Issue by thorellk** [tseemann/snippy#114]
Hi!

I am currently comparing > 1000 genomes to several references and am basically only going to use the output to create core genome alignments within different constellation of strains. Since keeping all output files take a lot of space it would be great to know what the minimum set of output files in each snippy directory would need to be to still be able to apply snippy-core to them. That way one can delete all other files while keeping only the possibility to rerun snippy-core if one wants to :)

Best wishes,

Kaisa

**Comment by tseemann** [comment]
Your question is very timely!

The only files you need to keep are
1. `snps.aligned.fa`
2. `snps.tab`

You used to have to need to keep the `reference/` folder too, but I have removed that requirement and added a `--ref` option to `snippy-core` where you must give it a FASTA file of the reference with the exact same IDs. You can get it from one of the snippy folders before you delete them all.

**Comment by thorellk** [comment]
Thank you!

**Comment by thorellk** [comment]
Hi!
Has this requirement changed? I was just going to calculate the core genome of a very large number of genomes so I removed all what I thought were unnecessary files so that the guys hosting our HPC would not be angry at me for creating a gzillion files. But now when I ran snippy-core I got "ERROR: Could not read VCF 'VIB_AA0002AA_AS/snps.vcf'". Do I also need to save the vcf files and are there other files that are required as well?
Best wishes,
Kaisa
